tests/helper.cpp

```diff
@@ -1,3 +1,11 @@
+// SPDX-FileCopyrightText: 2020-2023 Ravil Dorozhinskii
+//
+// SPDX-License-Identifier: MIT
+
 #include "helper.hpp"
+#include "FtyDataTypes.hpp"
+#include <utility>
 
-BlockT make_block(StringsT &Content) { return std::make_pair(Content.begin(), --Content.end()); }
\ No newline at end of file
+auto make_block(StringsT& Content) -> BlockT {
```

Comment:
Why replace the return type with auto and then give it a trailing return type here?